games-engines/exult/exult-1.6.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="an Ultima 7 game engine that runs on modern operating systems"
+HOMEPAGE="http://exult.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/exult-all-versions/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="alsa opengl +sdl2 fluidsynth timidity tools zlib"
+
+DEPEND="
+	games-misc/exult-sound
+	>=media-libs/libpng-1.6:0=
+	media-libs/libvorbis
+	alsa? ( media-libs/alsa-lib )
+	fluidsynth? ( media-sound/fluidsynth )
+	opengl? ( virtual/opengl )
+	sdl2? ( media-libs/libsdl2[sound,video,X] )
+	!sdl2? ( media-libs/libsdl[sound,video,X] )
+	timidity? ( >=media-sound/timidity++-2 )
+	zlib? ( sys-libs/zlib )
+"
+RDEPEND="${DEPEND}"
+
+DOCS=(
+	AUTHORS ChangeLog FAQ NEWS README README.1ST
+)
+
+src_configure() {
+	econf \
+		--enable-mods \
+		--with-desktopdir=/usr/share/applications \
+		--with-icondir=/usr/share/pixmaps \
+		--with-sdl=$(usex sdl2 sdl2 sdl12) \
+		$(use_enable alsa) \
+		$(use_enable fluidsynth) \
+		$(use_enable opengl) \
+		$(use_enable timidity timidity-midi) \
+		$(use_enable tools) \
+		$(use_enable zlib zip-support)
```

Comment:
Actually this does **not** compile with `-zlib`, although the error doesn't say anything about zlib. Could you take a look? Patch it or make it unconditional dep.
```
gamedat.cc:93:6: error: no declaration matches 'void Game_window::restore_flex_files(IDataSource&, const char*)'
   93 | void Game_window::restore_flex_files(
      |      ^~~~~~~~~~~
gamedat.cc:93:6: note: no functions named 'void Game_window::restore_flex_files(IDataSource&, const char*)'
In file included from gamedat.cc:42:
gamewin.h:94:7: note: 'class Game_window' defined here
   94 | class Game_window {
      |       ^~~~~~~~~~~
gamedat.cc: In member function 'void Game_window::restore_gamedat(const char*)':
gamedat.cc:231:2: error: 'restore_flex_files' was not declared in this scope
  231 |  restore_flex_files(in, GAMEDAT);
      |  ^~~~~~~~~~~~~~~~~~
```